lts_functions.py

```diff
@@ -1,209 +1,188 @@
-import re
-import yaml
-
+# import pytest
 import pandas as pd
 import numpy as np
+import re
 
-# %% Read Configuration Files
-def read_tables():
-    with open('config/tables.yml', 'r') as yml_file:
-        tables = yaml.safe_load(yml_file)
-    return tables
-
-def read_rating():
-    with open('config/rating_dict.yml', 'r') as yml_file:
-        rating_dict = yaml.safe_load(yml_file)
-    return rating_dict
-
-# %% Reused Functions
-
-def apply_rules(gdf_edges, rating_dict, prefix):
-    rules = {k:v for (k,v) in rating_dict.items() if prefix in k}
-        
-    for key, value in rules.items():
-        # Check rules in order, once something has been updated, leave it be
-        # FIXME gracefully handle if condition is not found
-        # FIXME need to handle single sided tags so that can include both sides in outputs
-        # print(key)
-        try:
-            gdf_filter = gdf_edges.eval(f"{value['condition']} & (`{prefix}_condition` == 'default')")
-            gdf_edges.loc[gdf_filter, prefix] = value[prefix]
-            gdf_edges.loc[gdf_filter, f'{prefix}_rule_num'] = key
-            gdf_edges.loc[gdf_filter, f'{prefix}_rule'] = value['rule_message']
-            gdf_edges.loc[gdf_filter, f'{prefix}_condition'] = value['condition']
-            if 'LTS' in value:
-                gdf_edges.loc[gdf_filter, f'LTS_{prefix}'] = value['LTS']
-        except pd.errors.UndefinedVariableError as e:
-            print(f'Column used in condition does not exsist in this region:\n\t{e}')
-
-    # Save memory by setting as category, need to set categories first
-    for col in [
-        # prefix,
-        f'{prefix}_rule_num',
-        f'{prefix}_condition', 
-        f'{prefix}_rule',
-        ]:
-
-        gdf_edges[col] = gdf_edges[col].astype('category')
-
-    return gdf_edges
-
-def convert_feet_with_quotes(series):
-    series = series.copy()
-    # Calculate decimal feet and inches when each given separately
-    quoteValues = series.str.contains('\'')
-    meterValues = quoteValues == False
-
-    quoteValues[quoteValues.isna()] = False
-    quoteValues = quoteValues.astype(bool)
-
-    feetinch = series[quoteValues].str.strip('"').str.split('\'', expand=True)
-    if feetinch.shape[0] > 0:
-        feetinch.loc[feetinch[1] == '', 1] = 0
-        feetinch = feetinch.apply(lambda x: np.array(x, dtype = 'int'))
-    # if feetinch.shape[0] > 0:
-        feet = feetinch[0] + feetinch[1] / 12
-        series[quoteValues] = feet
-
-    # Use larger value if given multiple
-    multiWidth = series.str.contains(';', na=False) 
-
-    maxWidth = series[multiWidth].str.split(';', expand=True).fillna(value=np.nan).astype(float).max(axis=1)
-    series[multiWidth] = maxWidth
-
-    series = pd.to_numeric(series, errors='coerce')
-    # series = series.apply(lambda x: np.array(x, dtype = 'float'))
-
-    # Convert (assumed) meter values to feet
-    series[meterValues] = series[meterValues].astype(float) * 3.28084
+# unit = 'metric'
```

Comment:
looks like much of this is being reverted from the work I did

Reply:
I made a new commit that tried to revert this. I only saw your tests on the bottom that you added, let me know if you did more than that